src/applications/rated-disabilities/containers/App.jsx

```diff
@@ -11,18 +11,16 @@ import { isLoggedIn } from '@department-of-veterans-affairs/platform-user/select
 import environment from '@department-of-veterans-affairs/platform-utilities/environment';
 
 import AppContent from '../components/AppContent';
-import FeatureFlagsLoaded from '../components/FeatureFlagsLoaded';
 import MVIError from '../components/MVIError';
-import { isLoadingFeatures } from '../selectors';
 import { useBrowserMonitoring } from '../util/datadog-rum/useBrowserMonitoring';
 
 const App = props => {
-  const { featureFlagsLoading, loggedIn, user } = props;
+  const { loggedIn, user } = props;
 
   // Add Datadog UX monitoring to the application
   useBrowserMonitoring({
     loggedIn,
-    version: '1.0.0',
+    version: '1.0.3',
```

Comment:
I set this to 1.0.3 instead of 1.0.1 because when I was testing this locally I used 1.0.1 and 1.0.2 to test out some different configuration options, using 1.0.3 should allow us to evaluate changes without having to worry about those sessions interfering